.github/workflows/release-client-api.yml

```diff
@@ -0,0 +1,65 @@
+name: Release api package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+          registry-url: https://npm.pkg.github.com/
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Install dependencies
+        run: |
+          cd tee-worker/client-api
+          corepack yarn install
+      - name: Build and Publish Parachain
+        run: |
+          branch_name="${{ github.ref }}"
+          echo "Branch Name: $branch_name" 
+          if [ "$branch_name" == "refs/heads/dev" ]; then
+            cd tee-worker/client-api/parachain-api
+            corepack yarn build
```

Comment:
How do we ensure here that the worker is running (with the expected ports)?
For reference, the ts-tests use docker-compose to set, e.g.: `depends_on.litentry-worker-1.condition: service_healthy`; see `tee-worker/docker/lit-di-vc-test.yml`

Reply:
Emmm, I'm directly using metadata to build here (assuming that the metadata on the branch is correct). If follow your suggestion, do we need to update the metadata and then build?

Reply:
Sort of. The ultimate goal is to get rid of the committed metadata and use a fresh one each time we build the package; see #2092